packages/plugin-slider/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "plugin-slider",
+  "version": "0.0.1",
+  "description": "Will provide a slider functionality in order to begin experiments      ",
```

Comment:
This should be the full description -- it'll eventually get auto-doc'd from this.